commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java

```diff
@@ -67,9 +68,15 @@ public CPVersionContributor getCPVersionContributor(String key) {
 			cpVersionContributorServiceWrappers = ListUtil.fromCollection(
 				_serviceTrackerMap.values());
 
-		List<CPVersionContributor> cpVersionContributors = ListUtil.toList(
-			cpVersionContributorServiceWrappers,
-			ServiceWrapper<CPVersionContributor>::getService);
+		List<CPVersionContributor> cpVersionContributors = new ArrayList<>();
```

Comment:
We could also initialize this list with the size of the `cpVersionContributorServiceWrappers`, for a little more efficiency.